From 51bcc2cb77336c94076f0267e9e3ab9880a47e54 Mon Sep 17 00:00:00 2001
From: James Greig <3stripe@gmail.com>
Date: Sat, 7 Mar 2026 12:00:31 +0000
Subject: [PATCH 05/32] E022 S02: update modules-private submodule
 (restoreSafeMode removed)

---
 modules-private | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules-private b/modules-private
index 0f6b838..c4357e6 160000
--- a/modules-private
+++ b/modules-private
@@ -1 +1 @@
-Subproject commit 0f6b8382a4ea4d189ed0468cf3729b954a719c30
+Subproject commit c4357e66bd8d8e9bff03f24e9be1ba0baf1b2ac9
-- 
2.39.5 (Apple Git-154)

